controllers/policy_utils.go

```diff
@@ -140,8 +140,8 @@ func reconcilePolicy(ctx context.Context, client client.Client, reconciler admis
 			Message: "The latest replica set is uniquely reachable",
 		},
 	)
-	policy.SetStatus(v1alpha2.PolicyStatusActive)
 
+	policy.SetStatus(v1alpha2.PolicyStatusActive)
```

Comment:
I think it would be better to put the policy in active state right after the webhooks have been reconciled